### PR TITLE
Curiesync ipinfo

### PR DESCRIPTION
**Pull request by yoavkatzman**

### CODE REVIEW

File: curiefense/images/curiesync/init/pull.sh [hunk 1]
```diff
@@ -45,6 +47,7 @@ if [ "$RUN_MODE" = "PERIODIC_SYNC" ] || [ -z "$RUN_MODE" ]; then
     do
         info "Pulling ${CURIE_BUCKET_LINK}"
         curieconfctl sync pull "${CURIE_BUCKET_LINK}" /cf-config
+        curieconfctl sync pullipinfo rbz-internal rbz-dev-auto-acl ipinfo /cf-config/ipinfo/
```
Comment (Freakachoo) [inline curiefense/images/curiesync/init/pull.sh:50]: Is that fine that we have here hardcoded `rbz-dev-auto-acl` don't we want per prod/dev bucket?

File: curiefense/curieconf/client/curieconf/cli/__init__.py [hunk 2]
```diff
@@ -460,6 +474,43 @@ def push(source_path: str, target_url: str, version: str = ""):
...
+            else:
+                logger.info(f"{target_path} already contains the updated {file_name}")
+
+
+
+
+
+
+
+
```
Comment (Freakachoo) [inline curiefense/curieconf/client/curieconf/cli/__init__.py:513]:
```suggestion


```

File: curiefense/images/curiesync/init/pull.sh (hunk 1 above)
Comment (aisenbergmor) [reply curiefense/images/curiesync/init/pull.sh:52]: for now yes, next phase we will add variable for it to be more dynamic

File: curiefense/images/curiesync/init/pull.sh [hunk 3]
```diff
@@ -20,6 +20,8 @@ fi
 if [ "$RUN_MODE" = "SYNC_ONCE" ]; then
     info "Synchronizing once"
     curieconfctl sync pull "${CURIE_BUCKET_LINK}" /cf-config
+    curieconfctl sync pullipinfo rbz-internal rbz-dev-auto-acl ipinfo /cf-config/ipinfo/
```
[the remark this answers is not in the document]
Comment (aisenbergmor) [reply curiefense/images/curiesync/init/pull.sh:23]: for now yes, next phase we will add variable for it to be more dynamic

**Review by tzuryby — APPROVED** [review]

**Pull request merged**